fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/election/ElectionVote.java

```diff
@@ -3,25 +3,19 @@
 import com.github.dedis.popstellar.model.objects.Election;
 import com.google.gson.annotations.SerializedName;
 
-import java.util.Arrays;
-import java.util.List;
-
-public class ElectionVote {
+public class ElectionVote{
 
+  @SerializedName(value = "id")
   private final String id;
-  /**
-   * Id of the object ElectionVote : Hash(“Vote”||election_id|| ||
-   * question_id||(vote_index(es)|write_in))
-   */
+
+   // Id of the object ElectionVote : Hash(“Vote”||election_id|| || JsonTestUtils.testData(castOpenVote);
   @SerializedName(value = "question")
   private final String questionId; // id of the question
 
-  private final List<Integer> vote; // list of indexes for the votes
-  private final boolean
-      writeInEnabled; // represents a boolean to know whether write_in is allowed or not
-
-  @SerializedName(value = "write_in")
-  private final String writeIn; // If write in is enabled this represents the writeIn string
```

Comment:
Will this evict the `write_in `field from the json? Could fail json schema validation on the backend potentially since the `write_in` is in the specifications for now. A solution is to keep it and not use it or update the specification and ensure this does not cause problems with the json validator on the backend

Reply:
The write in is only used while encrypting the hash. If i use it, it will create problem while I serialize my class.  I think they didnt want to keep it in the protocol since it is probably not a big priority.

Reply:
I agree, write_in doe snot bring anything in the current state of the project bit is still checked in the validator of json schemas on the backends and if write_in is not find any cast vote message will not adhere to the schema and will be rejected by the backend right? If you already addressed this even better, but than the specs should be updated

Reply:
Currently, the Json scheme doesnt need the write in, so the backend shouldnt reject it.

Reply:
That's good news!

Reply:
i mark it as resolved